CMake/stb/blib.hpp

```diff
@@ -0,0 +1,16 @@
+
+#ifndef BLIB_HPP
+#define BLIB_HPP
+
+#define STB_IMAGE_STATIC
+#define STB_IMAGE_IMPLEMENTATION
```

Comment:
I think that including `stb_image.h` and these definitions need to go in the .cpp files---if they're in the headers, then `stb_image.h` won't be included twice because of its header guard.